app/policies/user_policy.rb

```diff
@@ -0,0 +1,62 @@
+class UserPolicy < BasePolicy
+
+  def new?
+    # invitations#new
+    current_user.superadmin? || current_user.admin? || current_user.organisation_admin?
+  end
+  alias_method :index?, :new?
+
+  def edit?
+    return current_user.superadmin? if record.api_user?
```

Comment:
this line was left behind after i wrote a separate `ApiUserPolicy`. i've taken it out.